pages/dkp/kommander/2.2/dkp-upgrade/upgrade-kommander/index.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: layout.pug
+navigationTitle: Upgrade Kommander
+title: Upgrade Kommander
+menuWeight: 20
+excerpt: Steps to upgrade Kommander via CLI
+beta: false
+enterprise: true
+---
+ 
+This section describes how to upgrade your Kommander Management cluster and all its Platform Applications to the latest versions. Kommander is installed on your Management cluster and upgrading it must be the first step you take when upgrading DKP, to prevent compatibility issues.
+
+<p class="message--note"><strong>NOTE: </strong>It is important to upgrade Kommander BEFORE upgrading the Kubernetes version (or Konvoy version for Managed Konvoy clusters) in attached clusters, due to the previous versions' incompatibility with 1.22.</p>
+
+## Prerequisites
+
+- [Download][download-binary] and install the latest DKP CLI binary on your computer.
+- Ensure you are on DKP version 2.1 or 2.1.1 and Kubernetes version 1.21.
+- If you have attached clusters, ensure they are on Kubernetes versions 1.19, 1.20 or 1.21. To upgrade your Kubernetes version, refer to [AKS][AKS], [AWS][AWS], [Azure][Azure], [EKS][EKS], [pre-provisioned][pre-provisioned] documentation.
+- Review the [Platform Application version updates][platform-apps] that are part of this upgrade.
+
+## Upgrade Kommander
+
+1.  Use the DKP CLI to upgrade Kommander and all the Platform Applications in the Management cluster:
```

Comment:
@cneth you commented it might be a good idea to ensure customers have their KUBECONFIG environment variable pointing at the Management cluster? How about this?